include/gfxbase.h

```diff
@@ -1132,41 +1128,40 @@ class GFXBase : public Adafruit_GFX
 
     void BresenhamLine(int x0, int y0, int x1, int y1, CRGB color, bool bMerge = false) const
     {
-        int dx = abs(x1 - x0), sx = x0 < x1 ? 1 : -1;
-        int dy = -abs(y1 - y0), sy = y0 < y1 ? 1 : -1;
+        int dx = abs(x1 - x0); // Delta in x direction
+        int dy = abs(y1 - y0); // Delta in y direction
+        int sx = (x0 < x1) ? 1 : -1; // Step in x direction
+        int sy = (y0 < y1) ? 1 : -1; // Step in y direction
 
-        int err = dx + dy;  // Error must be declared here
+        int err = dx - dy; // Initial error term
 
-        for (;;)
+        while (true)
         {
-            if (isValidPixel(x0, y0))
-                leds[XY(x0, y0)] = bMerge ? leds[XY(x0, y0)] + color : color;
+            int index = XY(x0, y0);
+            if (isValidPixel(index))
+            {
+                // Optimization opportunity: unswtitch bMerge into another function
+                leds[index] = bMerge ? leds[index] + color : color;
+            }
 
             if (x0 == x1 && y0 == y1)
-                break;
+                break; // Exit the loop once we've reached the destination
 
-            int e2 = 2 * err;
-            if (e2 >= dy)
+            int e2 = 2 * err; // Error term multiplied by 2 for efficiency. Saves second test for Y.
+            if (e2 > -dy) // Move in the x direction if needed
```

Comment:
Are you sure about the switch from "greater or equal" to "greater than"? Same question for the "less than" change 7 lines down. Phrased differently and actually more to the point: why _should_ this be changed?